airbyte-db/lib/src/main/java/io/airbyte/db/DataTypeUtils.java

```diff
@@ -37,6 +40,10 @@
     }
   }
 
+  public static String toISO8601StringWithMilliseconds(final long epochMillis) {
+    return DATE_FORMAT_WITH_MILLISECONDS.format(Date.from(Instant.ofEpochMilli(epochMillis)));
```

Comment:
from task definition: _The cursor should capture the exact data with microseconds_
Not milliseconds